### PR TITLE
Lintrunner compiler warning patch

### PR DESCRIPTION
**Pull request by jjsjann123**
Enable lintrunner on third_party/nvfuser/
Patching compiler warning on un-used variable

### CODE REVIEW

File: .lintrunner.toml [hunk 1]
```diff
@@ -54,6 +54,10 @@ include_patterns = [
     'torch/csrc/**/*.cpp',
     'test/cpp/**/*.h',
     'test/cpp/**/*.cpp',
+    'third_party/nvfuser/csrc/**/*.h',
+    'third_party/nvfuser/csrc/**/*.cpp',
+    'third_party/nvfuser/test/**/*.h',
+    'third_party/nvfuser/test/**/*.cpp',
```
Comment (naoyam) [inline .lintrunner.toml]: Why not the benchmark files?

**Review by naoyam — APPROVED** [review]
LGTM

File: .lintrunner.toml (hunk 1 above)
Comment (jjsjann123) [reply .lintrunner.toml]: good question. they were not applied to benchmark files originally... But would be a nice addition~~

**Comment by jjsjann123** [comment]
Ha, running clangformat has bumped the LOC up quite a lot. But they are just format changes and I have verified the build locally (with benchmark built as well).
I'll merge this once CI passed.

**Comment by jjsjann123** [comment]
built worked for me. even though I don't currently have a machine to verify tests...
I'm gonna be bold and merge it since format changes shouldn't break anything at runtime.... :crossed_fingers:

**Pull request merged**